libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -187,6 +187,7 @@ - (id)init {
         _authPreferences = [SFSDKAuthPreferences  new];
         _errorManager = [[SFSDKAuthErrorManager alloc] init];
         _shouldFallbackToWebAuthentication = NO;
+        _showAuthWindowWhileLoading = NO;
```

Comment:
I can't remember if we made a decision about what the default behavior should be, I can flip this and have consumers opt out if they want as well

Reply:
This is fine IMO.  If we thing it is safe we can always flip the default in the next major.